awx/main/dispatch/worker/base.py

```diff
@@ -127,10 +128,20 @@ class AWXConsumerRedis(AWXConsumerBase):
     def run(self, *args, **kwargs):
         super(AWXConsumerRedis, self).run(*args, **kwargs)
         self.worker.on_start()
-
+        subsystem_metrics = s_metrics.Metrics(auto_pipe_execute=False)
+        last_debug_time = 0
+        previous_requests = 0
         while True:
-            logger.debug(f'{os.getpid()} is alive')
-            time.sleep(60)
+            requests = subsystem_metrics.update_uwsgi_stats()
```

Comment:
we need somewhere to call this update_uwsgi_stats() method. The problem with doing it in `callback.py` is that there are mulitiple forks of the callback receiver, and calling this method in each of those could be expensive and wasteful.

So I found this `while True` loop that runs the in the parent thread of all those callback receiver workers. Not ideal, but it does work.